Assets/UI/Panels/citypanel.lua

```diff
@@ -1469,6 +1612,27 @@ function OnCityMadePurchase(owner:number, cityID:number, plotX:number, plotY:num
     end
 end
 
+
+-- ==== CQUI CUSTOMIZATION BEGIN ====================================================================================== --
+-- CQUI does not implement this function (unmodified defines this function here)
+-- function OnProductionPanelListModeChanged( listMode:number )
+-- ==== CQUI CUSTOMIZATION END ======================================================================================== --
+
+-- ===========================================================================
```

Comment:
these next two functions match the unmodified firaxis code and match what had previously been declared as inline functions in the initialize method